### PR TITLE
Updating Translations 2003 2/26/2020

### PR DESCRIPTION
**Pull request by BrightspaceGitHubReader**

### CODE REVIEW

File: lang/ko.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-    "noOutcomesStudent": "No {outcome, select, competencies {competencies} expectations {expectations} objectives {objectives} outcomes {outcomes} standards {standards} other {standards}} have been evaluated.",
-    "noScaleInstructor": "To view learner progress towards achieving this {outcome, select, competencies {competency} expectations {expectation} objectives {objective} outcomes {outcome} standards {standard} other {standard}}, create and select an achievement scale.",
-    "noScaleStudent": "No scale has been set to measure achievement of {outcome, select, competencies {competencies} expectations {expectations} objectives {objectives} outcomes {outcomes} standards {standards} other {standards}}.",
-    "notAssessed": "평가되지 않음",
-    "trend": "추세",
-    "untitled": "제목 없음"
+   "bigTrendAttemptsScreenReaderString" : "{numAttempts, plural, =1 {Attempt {attemptNames}} 기타 {Attempts {attemptNames} 및 {lastAttemptName}}}",
```
Comment (rgoessl) [inline lang/ko.json]:
```suggestion
   "bigTrendAttemptsScreenReaderString" : "{numAttempts, plural, =1 {Attempt {attemptNames}} other {Attempts {attemptNames} 및 {lastAttemptName}}}",
```

File: lang/ko.json [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+   "bigTrendAttemptsTooltipString" : "{numAttempts, plural, =1 {Attempt} 기타 {Attempts}} {attemptNames}",
```
Comment (rgoessl) [inline lang/ko.json]:
```suggestion
   "bigTrendAttemptsTooltipString" : "{numAttempts, plural, =1 {Attempt} other {Attempts}} {attemptNames}",
```

File: lang/ko.json [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+   "close" : "닫기",
+   "evidence" : "근거",
+   "headingDate" : "날짜",
+   "headingEvidence" : "근거 이름",
+   "headingLoa" : "성취 수준",
+   "miniTrendAttemptsTooltipString" : "{numAttempts, plural, =1 {Attempt} 기타 {Attempts}} {attemptNames}",
```
Comment (rgoessl) [inline lang/ko.json]:
```suggestion
   "miniTrendAttemptsTooltipString" : "{numAttempts, plural, =1 {Attempt} other {Attempts}} {attemptNames}",
```

File: lang/ko.json [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
+   "miniTrendScreenReaderText" : "평가됨 {numAssessed} {numAssessed, plural, =1 {time} 기타 {times}}: {levelNames}",
```
Comment (rgoessl) [inline lang/ko.json]:
```suggestion
   "miniTrendScreenReaderText" : "평가됨 {numAssessed} {numAssessed, plural, =1 {time} other {times}}: {levelNames}",
```

**Comment by sarahboyd** [comment]
> They messed up korean

when do they not?

**Pull request merged**